webpack.config.js

```diff
@@ -18,10 +19,14 @@ module.exports = (env, argv) => {
     devtool: prodSourceMaps ? 'source-map' : false,
     entry: !env.espOnly
       ? {
-          main: {
```

Comment:
`main` is used for single entry point, docs: https://webpack.js.org/concepts/entry-points/